src/models/LottoResult.js

```diff
@@ -0,0 +1,58 @@
+import { getInterSectionSize } from '../utils/intersection.js';
+
+class LottoResult {
+  #winLotto = new Map();
+  #winningNumbers;
+  #bonusNumber;
+
+  constructor(lottos, winningNumbers) {
+    this.#winningNumbers = winningNumbers.getWinningNumbers();
+    this.#bonusNumber = winningNumbers.getBonusNumber();
+    this.#initWinLotto();
+    this.#checkAllLottos(lottos);
+  }
+
+  getWinLotto() {
+    return this.#winLotto;
+  }
+
+  #initWinLotto() {
+    this.#winLotto.set(3, 0);
+    this.#winLotto.set(4, 0);
+    this.#winLotto.set(5, 0);
+    this.#winLotto.set('bonus', 0);
+    this.#winLotto.set(6, 0);
+  }
```

Comment:
```suggestion
this.#winLotto = new Map([[3, 0], [4, 0], [5, 0], ['bonus', 0], [6, 0]]);
```
함수 대신에 이렇게 간단히 나타낼 수 있지 않을까!  생각합니다

Reply:
헉 그러네요..!! 훨씬 보기 깔끔하군요 👍